pkgs/top-level/all-packages.nix

```
@@ -14660,12 +14660,8 @@ with pkgs;
 
   autoadb = callPackage ../misc/autoadb { };
 
-  inherit (callPackage ../tools/admin/ansible { })
-    ansible
-    ansible_2_8
-    ansible_2_9
-    ansible_2_10
-    ansible_2_11;
+  ansible = ansible_2_12;
+  ansible_2_12 = python3Packages.toPythonApplication python3Packages.ansible-core;
```

```suggestion
  ansible = python3Packages.toPythonApplication python3Packages.ansible-core;
```
Might as well. I don't want to support multiple ansible versions in the future if possible.

---

I like (and need) the option to have multiple versions of Ansible.  However only those that are supported upstream make sense.  Ansible will be officially deprecating 2.9.x next month (in May 2022): https://groups.google.com/g/ansible-announce/c/kegIH5_okmg/?pli=1.  So dropping ansible 2.9 and 2.10 make sense, while 2.11 and 2.12 are actively supported and used in the wild.

---

but neither me nor mweinelt want to maintain more versions than necessary.

---

Tracking multiple ansible versions is not unusual, so I will keep this as is.

With 2.11 I decided to drop it, because we can't support it throughout the 22.05 release.